plugins/modules/aci_interface_config.py

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_config
+short_description: Manage the Port Configuration of the Fabric Access Policies - Interface Configuration (infra:PortConfig)
+description:
+- Manage the Port Configuration of the Fabric Access Policies - Interface Configuration on Cisco ACI fabrics.
+options:
+  policy_group:
+    description:
+    - The Name of the Policy Group.
+    type: str
+    aliases: [ policy_group_name ]
+  breakout:
+    description:
+    - The Breakout of the interface.
+    - The policy_group should be empty while configuring the Breakout Map.
+    type: str
+    choices: [ 100g-2x, 100g-4x, 10g-4x, 25g-4x, 50g-8x ]
+  description:
+    description:
+    - Description of the Interface Configuration object.
+    type: str
+    aliases: [ descr ]
+  node:
+    description:
+    - The ID of the Node.
+    - The value must be between 101 to 4000.
+    type: int
+    aliases: [ node_id ]
+  pc_member:
+    description:
+    - The name of the Port Channel Member.
+    type: str
+    aliases: [ port_channel_member ]
+  port_type:
+    description:
+    - The type of the interface can be either access or fabric.
+    - The default port_type is access.
```

Comment:
Is this line needed when the default is already specified in the documentation 2 lines below?